app/src/main/kotlin/com/simplemobiletools/calculator/activities/MoneyActivity.kt

```diff
@@ -2,29 +2,24 @@ package com.simplemobiletools.calculator.activities
 
 import android.annotation.SuppressLint
 import android.app.AlertDialog
-import android.content.Context
 import android.content.res.ColorStateList
 import android.graphics.Color
 import android.os.Bundle
 import android.support.v4.view.ViewCompat
-import android.util.Log
 import android.view.View
 import android.widget.AdapterView
 import android.widget.ListView
 import com.simplemobiletools.calculator.R
-import com.simplemobiletools.calculator.extensions.config
-import com.simplemobiletools.calculator.extensions.updateViewColors
-import com.simplemobiletools.calculator.helpers.Calculator
+import com.simpletools.calculator.commons.extensions.config
```

Comment:
What is this change ?

Reply:
the config has moved to the commons module but it is used in the app module for the phone. The import path is therefore different